kura/org.eclipse.kura.core/OSGI-INF/metatype/org.eclipse.kura.data.DataService.xml

```diff
@@ -98,7 +98,36 @@
             default="0"
             min="0"
             description="Timeouts the in-flight messages congestion condition. The service will force a disconnect attempting to reconnect (0 to disable)."/>
-            
+
+        <AD id="average.publish.rate"
+            name="Average publish rate"
+            type="Integer"
+            cardinality="0"
+            required="true"
+            default="0"
+            min="0"
+            description="The average message publish rate. The unit of measurement is set by the publish rate unit of measurement option. If set to 0 the rate limit is disabled."/>
+        
+        <AD id="average.publish.rate.unit"
```

Comment:
I'll remove the unit option and set the rate as messages/second. I'll also update the description to give an example on how to use it. However, I don't like it.